Headlines/src/DataSources/FilterSourcesDataSource.swift

```diff
@@ -98,4 +98,19 @@ class FilterSourcesDataSource: NSObject, UICollectionViewDelegate, UICollectionV
         
         selectedSources.remove(at: deselectedIndex)
     }
+    
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        sizeForItemAt indexPath: IndexPath) -> CGSize {
+        
+        guard let cell = self.collectionView(collectionView, cellForItemAt: indexPath) as? FilterCollectionViewCell else {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/12563924cda50bbe12cad8cbfdd4405460c87ecd/Headlines/src/DataSources/FilterSourcesDataSource.swift#L106"></span>Line should be 120 characters or less: currently 122 characters</td>
    </tr>
  </tbody>
</table>